kraken-std/src/kraken/std/cargo/__init__.py

```diff
@@ -252,7 +252,7 @@ def cargo_bump_version(
     version: str,
     revert: bool = True,
     name: str = "cargoBumpVersion",
-    group: str | None = CARGO_PUBLISH_SUPPORT_GROUP_NAME,
```

Comment:
I think the task still needs to belong to `CARGO_PUBLISH_SUPPORT_GROUP_NAME` (you can add `task.depends_on(f":{CARGO_BUILD_SUPPORT_GROUP_NAME}?")` after the task has been created)